<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_SetPlayerGroupClass</class>
 <widget class="QWidget" name="Dlg_SetPlayerGroupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change player group</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Select player group</string>
   </property>
  </widget>
  <widget class="MoveListWidget" name="listItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget
{
    background: transparent;
    border: none;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveGroup">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save group</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MoveListWidget</class>
   <extends>QListWidget</extends>
   <header>MoveListWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnCancel</tabstop>
  <tabstop>groupTitle</tabstop>
  <tabstop>btnSaveGroup</tabstop>
  <tabstop>listItems</tabstop>
  <tabstop>btnOK</tabstop>
 </tabstops>
 <resources>
  <include location="../RC/Versus.qrc"/>
 </resources>
 <connections/>
</ui>
